src/sentry/api/endpoints/grouping_levels.py

```diff
@@ -115,18 +114,42 @@ def get_levels_overview(group):
 
     fields = res["data"][0]
 
+    if not fields["longest_hierarchical_hashes"]:
```

Comment:
empty array => regular grouping applies and we cannot show any sort of levels information or anything based on it